packages/jazz-auth-clerk/src/index.ts

```diff
@@ -38,13 +38,18 @@ export class JazzClerkAuth {
   }
 
   onClerkUserChange = async (clerkClient: Pick<MinimalClerkClient, "user">) => {
+    const jazzIsAuthenticated = this.authSecretStorage.isAuthenticated;
+
     if (!clerkClient.user) {
+      if (jazzIsAuthenticated) {
```

Comment:
We need to ensure that if the user is already authenticated id doesn't get logged out during the load.

Reason: clerkClient.user might be undefined because we are still loading clerk.

Also this change affects both Web and Native, and it needs to be tested on both.